src/components/Transparency/BudgetCard.tsx

```diff
@@ -0,0 +1,34 @@
+import { useIntl } from 'react-intl'
+
+import { SubtypeAlternativeOptions } from '../../entities/Grant/types'
+import { CURRENCY_FORMAT_OPTIONS } from '../../helpers'
+import useBudgetByCategory from '../../hooks/useBudgetByCategory'
+import useFormatMessage from '../../hooks/useFormatMessage'
+import Time from '../../utils/date/Time'
+import locations from '../../utils/locations'
+import { ProjectTypeFilter } from '../Search/CategoryFilter'
+import FinancialCard from '../Updates/FinancialCard'
+
+function BudgetCard() {
+  const t = useFormatMessage()
+  const { allocatedPercentage: percentage, allocated, total } = useBudgetByCategory(SubtypeAlternativeOptions.All)
+  const quarter = Time().quarter()
+  const { formatNumber } = useIntl()
+
+  return (
+    <div className="DaoFinancial">
```

Comment:
I prefer to create a new component `<DaoFinancial />` that imports the css than having the same class in one file used across components 🙏